pkg/util/trivy/containerd.go

```diff
@@ -0,0 +1,177 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
```

Comment:
Can you update the code owner for `pkg/util/trivy`

Reply:
Done :+1: